app/page.js

```diff
@@ -1,113 +1,196 @@
-import Image from 'next/image'
+import Image from "next/image";
+import homePageBanner from "./assets/images/homepage-banner.webp";
+import homePageImageTwo from "./assets/images/homepageimagetwo.webp";
+import homePageImageThree from "./assets/images/homepagethree.webp";
+import homePageImageFour from "./assets/images/homepagefour.webp";
+import fiverrWayImageOne from "./assets/images/fiverrwayimageone.webp";
+import fiverrWayImageTwo from "./assets/images/fiverrwayimagetwo.webp";
+import fiverrWayImageThree from "./assets/images/fiverrwayimagethree.webp";
+import fiverrWayImageFour from "./assets/images/fiverrwayimagefour.webp";
+import fiverrWayImageFive from "./assets/images/fiverrwayimagefive.webp";
+import fiverrWayImageSix from "./assets/images/fiverrwayimagesix.webp";
```

Comment:
create a common import export file for all images @PuniaMohit do not import like this
this should be like
import { homePageBanner, ....... } from '../../images'; @PuniaMohit